content/posts/2021-12-23-scm-manager-2-27-0/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: SCM-Manager 2.28.0
+date: 2021-12-23T10:00:00+0000
+author: René Pfeuffer
+categories:
+- scm-manager
+- release
+keywords:
+- accessibility
+- search
+- branch
+---
+
+Hey SCM-Manager Community,
+
+some time has passed sind 2.27.0, and the hassle around log4shell brought us some bugfix releases (though we
+use logback and not log4j, so we had less severe issues in SCM-Manager).
```

Comment:
Das in Klammern klingt komisch (oder ich verstehe es nicht richtig): "obwohl wir..., hatten wir weniger schwer wiegende Probleme"
```suggestion
some time has passed sind 2.27.0, and the hassle around log4shell brought us some bugfix releases (because we
use logback and not log4j, we had some less severe issues in SCM-Manager).
```